site/docs/v1/tech/themes/index.adoc

```diff
@@ -7,7 +7,7 @@ description: Learn how to theme and the FusionAuth login pages (including forgot
 
 == Overview
 
-FusionAuth themes allow you to customize the OAuth2 / OpenID Connect login pages and other user workflows such as forgot password. In FusionAuth you may create one to many themes and assign a theme per tenant so that you can customize the user experience for different users.
+FusionAuth themes allow you to customize the OAuth2 / OpenID Connect login pages and other user workflows such as forgot password. In FusionAuth you may create one to many themes and assign a theme per application so that you can customize the user experience for different users.
```

Comment:
```suggestion
FusionAuth themes allow you to customize the OAuth2 / OpenID Connect login pages and other user workflows such as forgot password. In FusionAuth you may create one to many themes and assign a theme per tenant or application so that you can customize the user experience for different users.
```